.env

```diff
@@ -0,0 +1,15 @@
+REACT_APP_local = true
```

Comment:
move to .env.template
and add a section in the README.md indicating the developer should create his own.

Reply:
Done!